src/main.cpp

```diff
@@ -194,13 +216,25 @@ int main(int argc, const char *argv[]) {
         logger, httpServer, tokenValidator, accessCheck);
     subHandler->addPublisher(mqttPublisher);
 
-    auto database =
-        std::make_shared<VssDatabase>(logger, subHandler);
-    auto cmdProcessor = std::make_shared<VssCommandProcessor>(
-        logger, database, tokenValidator, accessCheck, subHandler);
+    std::shared_ptr<VssDatabase> database = std::make_shared<VssDatabase>(logger,subHandler);
+
+    if(variables["record"].as<string>() == "recordSet" || variables["record"].as<string>()=="recordSetandGet")
+    {
+        if(variables["record"].as<string>() == "recordSet")
+          std::cout << "Recording inputs\n";
+        else if(variables["record"].as<string>() == "recordSetandGet")
```

Comment:
Is there any feature built for recordSetandGet? My lock file does not contain any get request after I start with the command: ` ./kuksa-val-server --record recordSetandGet`

Reply:
another point: if camelcase, pls always camelcase
```suggestion
        else if(variables["record"].as<string>() == "recordSetAndGet")
```